syntax = "proto3";

enum DeleteTarget
{
    DeleteAll = 0;
    DeleteSim = 1;
    DeleteLost = 2;
}


message hgDeleteTarget
{
    int32 DeleteType = 1;  //É¾³ıÀàĞÍ
}

